types/angular-agility/tslint.json

```diff
@@ -74,6 +74,7 @@
         "typedef-whitespace": false,
         "unified-signatures": false,
         "void-return": false,
-        "whitespace": false
+        "whitespace": false,
+        "no-angle-bracket-type-assertion": false
```

Comment:
For consistency, please don't turn off linter rules. Especially on so many packages at once.

[You can use `as` instead.](https://palantir.github.io/tslint/rules/no-angle-bracket-type-assertion/)

Reply:
The lint failures were caused by an unrelated change (https://github.com/Microsoft/dtslint/pull/141).

I really wasn't interested in fixing all of those failures by hand.

Reply:
Hmm. The tests fail when you revert these changes?

Reply:
Yes, they're still failing. 565 violations of the `no-angle-bracket-type-assertion` rule.

Reply:
Well I'm really sorry about that, I was unaware. That PR must have been merged without the necessary followup.

Thanks for fixing all the fallout. Could you revert your revert then? Once the tests pass I can merge this PR.